### PR TITLE
Jawn/delta

### PR DESCRIPTION
**Pull request by jawndiego**
changed ponder config to match updated ponder package
added isValidUser based on check of riverValidator status for user id before proceeding in noderegistry/register crud

### CODE REVIEW

**Review by 0xTranqui — CHANGES_REQUESTED** [review]
Screenshot 2023-11-15 at 11 12 07 AM

this looks great, and after seeing how u split out Access into its own entity, i think we can make the followign chagnes to the node entity to avoid duplicate data:

- take out msgType + msgBody from the Node entity, since those will be specific to messages. 
- what we want to add back in similar to how we can now derive access control for a given node, is to be able to derrive all messages that are sent to a given node. so adding something back into Node entity like:

`messages: [Message!]! @derivedFrom[field: message]

and then in the Message entity, update it so it has a field like:

node: Node! (which references the Node entity). this means we also need to add in NodeId tracking to the message entity

basically get it so with node as top level enttity, we can drill down into all down stream access control + messages. it might make sense for us to get rid of channel + publication entities, since they are techincally duplicate functionality that Node should service, with the ability to filter data by node schema giving us the ability to have views into channels + publications. lmk what u think
